.github/scripts/standup_lib.py

```
@@ -0,0 +1,292 @@
+    "payjoin/btsim",
+    "payjoin/tx-indexer",
+    "Uniffi-Dart/uniffi-dart",
+]
+
+REPO_FILTER = " ".join(f"repo:{r}" for r in REPOS)
+
+
+def graphql(query, variables=None):
+    """Run a GraphQL query with retry on 403 rate limits."""
+    for attempt in range(5):
+        resp = requests.post(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)